web/src/views/Home.vue

```diff
@@ -81,7 +81,7 @@
     }
 
     #discord {
-        $color: #7289DA;
+        $color: #687BBD;
```

Comment:
```suggestion
        $color: #7289DA;
```

The color in this PR is not Discord's official purple color. If a darker color is needed, we should use one of their official dark colors. 

![image](https://user-images.githubusercontent.com/5175705/86761847-8d5ac000-c046-11ea-9184-4ef2253b9275.png)

Source: https://discord.com/new/branding

Reply:
I know, but it was detected by validators such as web.dev as too bright for the text color, so it was the only way to achieve it right.